<?xml version="1.0" ?>
<launch>
  <arg name="namespace" default="/rasberry_coordination"/>

  <arg name="agent_id"/>
  <arg name="setup"/>

  <group ns="$(arg namespace)">
    <node pkg="rasberry_coordination" name="add_agent" type="add_agent.py" args="$(arg agent_id) $(arg setup)" output="screen"/>
  </group>

</launch>


